docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "2"
+
+services:
+  runner:
+    image: lubien/project_euler
+    command: python3 stats.py -b
+                              --s python
+                              --s commonlisp
+                              --s ruby
+                              --s haskell
+                              --s go
+                              --s php
+                              --s python
+                              --s elixir
+                              --s lua
+                              --s c++
```

Comment:
@lubien you can enable `C` tests for while just to testing if works now?

Reply:
I know you wanna fuck me there and didn't even took me to a date but ok